### ISSUE TITLE
Unable to edit JSON in IE11

### ISSUE THREAD

**Issue by jonlynch** [Esri/ago-assistant#166]
I have been unable to edit JSON in IE11. After some research [this thread](http://stackoverflow.com/questions/1391278/contenteditable-change-events) explains that the input event of contentEditable elements does not fire in IE10 or IE11. This stops the validation from firing. An example is at [http://jsfiddle.net/ch6yn/](http://jsfiddle.net/ch6yn/).

By changing `input` to `keyup` on [line #352 of main.js](https://github.com/Esri/ago-assistant/blob/master/src/js/main.js#L352) I was able to get the validation to fire and save updated json. 

I am not sure if you would rather detect IE and then add a number of other events such as `keyUp`, `paste` and `cut` but use `input` for all other browsers or change the event listener for all browsers? I have never submitted a pull request before but would be happy to make this my first one if given direction about the most appropriate solution.

**Comment by ecaldwell** [comment]
Thanks for reporting this. I'd prefer to just drop IE support entirely, but that's probably not a good approach. :bomb: 

I think another approach to addressing this is to manually trigger the validation when they hit the save button. IE users won't get any feedback while they edit, but it would at least give some feedback when trying to save (and let them proceed if it's all ok). I'd favor this approach over adding special logic to detect and handle IE cases. 

Thoughts?

**Comment by jonlynch** [comment]
I'm afraid that the company that I work for will be on IE only for a while yet 😭. This tool is used to copy (and update) content between development, test and production Portals. Whilst I can see your argument for not introducing code that targets any particular browser, given that the most recent ArcGIS Javascript API 4.x supports IE11 I would hope that IE11 could be supported as a first class citizen here as well.

I have opened a pull request #169 of the change I made and deployed at work (on IE11). Obviously I would be very happy if you merge it and so that we could benefit from all future development on this project. However I fully understand your argument, feel free to close it if you want.
